myapp/src/pages/Discussion/CreateBoard.js

```diff
@@ -0,0 +1,150 @@
+import React, {useState} from "react";
+import axios from "axios";
+//import {Redirect} from "react-router-dom";
+import Padding from "../../components/Padding";
+import {Form as BootStrapForm, Col} from "react-bootstrap";
+import {NewCard} from "./BoardElements.js";
+
+function CreateBoard(props) {
+  const [board, setBoard] = useState(
+    {
+      groupName: "",
+      name: "",
+      description: ""
+    }
+  );
+
+  function handleChange(event) {
```

Comment:
I would condense this function to the handleChange in the Filter for team roster
```
  function handleChange(event) {
    const { name, value } = event.target;
    const temp = { ...field };

    temp[name] = value;
    setField(temp);
  }
```